src/main/java/Storage.java

```diff
@@ -0,0 +1,105 @@
+package main.java;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.Task;
+import tasks.TaskList;
+import tasks.ToDo;
+import tasks.Deadline;
+import tasks.Event;
```

Comment:
I like how every imported class is listed explicitly instead of importing all at once using *.